src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -186,6 +193,66 @@ public function getColumnStyle($columnIndex)
         return $this->getStyle();
     }
 
+    /**
+     * Sets the width for a column.
+     *
+     * @param int        $columnIndex Column index.
+     * @param int|string $width       Column with in characters. Set to 'auto' to auto-fit the content.
+     *
+     * @return Table
+     */
+    public function setColumnWidth($columnIndex, $width)
+    {
+        $columnIndex = intval($columnIndex);
+
+        if ('auto' !== $width) {
+            $width = intval($width);
+        }
+
+        if ('auto' !== $width && 0 >= $width) {
```

Comment:
This first condition can be removed and the if statmeent moved into the previous if statement.